test/mobile/test_lite_script_module.py

```diff
@@ -431,5 +445,98 @@ def forward(self, val: int, x, y, w):
         self.assertTrue('test_lite_script_module.py\", line {}'.format(lineno + 8) in error_message)
         self.assertTrue('top(FooTest5)' in error_message)
 
+
+class TestLiteScriptQuantizedModule(QuantizationTestCase):
+
+    def _compare_script_and_mobile(self,
+                                   model_class: Type[torch.nn.Module],
+                                   input: torch.Tensor,
+                                   **kwargs):
+        qengine = "qnnpack"
+        with override_quantized_engine(qengine):
+            qconfig = torch.quantization.get_default_qconfig(qengine)
+            model = model_class(**kwargs)
+            # model.qconfig = qconfig
+            # model = prepare(model)
+
+            # test_only_eval_fn(model, self.calib_data)
+            # model = convert(model)
+            model = quantize(model, test_only_eval_fn, [self.calib_data])
+
+            script_module = torch.jit.script(model)
+            script_module_result = script_module(input)
+
+            buffer = io.BytesIO(script_module._save_to_buffer_for_lite_interpreter())
+            buffer.seek(0)
+            mobile_module = _load_for_lite_interpreter(buffer)
+
+            mobile_module_result = mobile_module(input)
+
+            torch.testing.assert_allclose(script_module_result, mobile_module_result)
+            mobile_module_forward_result = mobile_module.forward(input)
+            torch.testing.assert_allclose(script_module_result, mobile_module_forward_result)
+
+            mobile_module_run_method_result = mobile_module.run_method("forward", input)
+            torch.testing.assert_allclose(script_module_result, mobile_module_run_method_result)
```

Comment:
One issue we had is flaky: the test crashes very 2-3 times. Could you put these part in a loop and run it around 5 times? If there's signal of CI performance we could reduce the number of times.